target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -149,7 +150,6 @@ ipq40xx_setup_dsl()
 	ucidef_add_vdsl_modem "$annex" "$tone" "$xfer_mode"
 	ucidef_set_interface_wan "dsl0" "pppoe"
 }
-
```

Comment:
Another oops which shouldn't be here.

Reply:
yep, this was just an oops from resolving a conflict I think, will fix also

Reply:
fixed